lib/core/VersionManager.ts

```diff
@@ -186,8 +192,16 @@ export default class VersionManager implements IVersionManager {
     return transactionSelector;
   }
 
+  public getVersionMetadataByTransactionTime (blockchainTime: number): AbstractVersionMetadata {
```

Comment:
I am not sure if I should call it transaction time or blockchain time.

Reply:
In core it is all 'transactionTime'

Reply:
I agree that blockTime makes more sense - also mentioned [here](https://github.com/decentralized-identity/sidetree/pull/703#issuecomment-635916463)

Reply:
The rest of this file uses blockchain time... which is why I asked this question. Will keep it blockchain time unless someone objects.